src/routes/AppRoutes.tsx

```diff
@@ -4,16 +4,19 @@ import Home from '../pages/Home';
 import Login from '../pages/Login';
 import Signup from '../pages/Signup';
 import Dashboard from '../pages/Dashboard';
+import LandingPage from '../pages/LandingPage';
 
 const AppRoutes: React.FC = () => {
   return (
     <Router>
       <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/login" element={<Login />} />
-        <Route path="/signup" element={<Signup />} />
-        <Route path="/dashboard" element={<Dashboard />} />
-        {/* Add more routes as needed */}
+        <Route path="/" element={<Home />} >
+          <Route path="/landingpage" element={<LandingPage />} />
```

Comment:
landing page should be / which is the home technically.

Reply:
as agreed to let the home has nav and footer and redirect to Landing or Dashboard based on if user logged in or not